QueryEngine/CalciteAdapter.cpp

```diff
@@ -452,14 +457,20 @@ class CalciteAdapter {
     CHECK(expr_type.IsObject());
     const auto agg_kind = to_agg_kind(expr["agg"].GetString());
     const bool is_distinct = expr["distinct"].GetBool();
-    const auto operand = get_agg_operand_idx(expr);
-    const bool takes_arg{operand >= 0};
-    if (takes_arg) {
-      CHECK_LT(operand, static_cast<ssize_t>(scan_targets.size()));
+    const auto operands = get_agg_operand_idxs(expr);
+    std::shared_ptr<Analyzer::Expr> arg_expr{nullptr};
+    std::shared_ptr<Analyzer::Constant> err_rate_expr{nullptr};
+    if (operands.size() >= 1) {
+      CHECK_LT(operands[0], scan_targets.size());
+      arg_expr = scan_targets[operands[0]]->get_own_expr();
+      if (operands.size() == 2) {
+        CHECK_LT(operands[1], scan_targets.size());
+        err_rate_expr = std::dynamic_pointer_cast<Analyzer::Constant>(scan_targets[operands[1]]->get_own_expr());
+        CHECK(err_rate_expr != nullptr);
+      }
     }
-    const auto arg_expr = takes_arg ? scan_targets[operand]->get_own_expr() : nullptr;
     const auto agg_ti = get_agg_type(agg_kind, arg_expr.get());
-    return makeExpr<Analyzer::AggExpr>(agg_ti, agg_kind, arg_expr, is_distinct);
+    return makeExpr<Analyzer::AggExpr>(agg_ti, agg_kind, arg_expr, is_distinct, err_rate_expr);
```

Comment:
Feel free to pass a `nullptr` here and discard the rest of the changes to the file. This is an old adapter we've used for migrating to Calcite before having the full relational algebra model; we should delete it.